### PR TITLE
Add scripts to rule them all

### LINKED ISSUE #8: Create scripts to rule them all

### PR DESCRIPTION
**Pull request by Lycopersicum**
Closes #8

### CODE REVIEW

File: script/bootstrap [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+#
```
Comment (GiedriusM) [inline script/bootstrap]: Wrong license. Should be either updated to project license or removed.

File: script/bootstrap [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    fi
+
+    return $?
+}
+
+echo "==> Updating package lists..."
+update_repositories
+
+echo "==> Installing building tools..."
+install_build_tools
```
Comment (GiedriusM) [inline script/bootstrap:51]: bootstrap script should also install git submodules

File: script/setup [hunk 3]
```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+#
```
Comment (GiedriusM) [inline script/setup]: License issue

File: script/setup [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    eval "cd ${PROJECT_BUILD_DIR}"
+    echo "cmake ${cmake_arguments} ${PROJECT_ROOT_DIR}"
+    eval "cmake ${cmake_arguments} ${PROJECT_ROOT_DIR}"
+    make
+    cd -
+}
+
+eval "${SCRIPTS_DIR}/bootstrap"
+
+echo "==> Building punica..."
```
Comment (GiedriusM) [inline script/setup]: Shouldn't "Building punica" section call build_punica?

File: script/update [hunk 5]
```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
```
Comment (GiedriusM) [inline script/update]: Same license issue

**Review by GiedriusM — CHANGES_REQUESTED** [review]

**Review by taubel — APPROVED** [review]

**Review by GiedriusM — APPROVED** [review]

**Pull request merged**